task-generator/remote/main.go

```diff
@@ -125,55 +125,62 @@ if [ -e "/ssh/error" ]; then
   #no server could be provisioned
   cat /ssh/error
   exit 1
+fi
+export SSH_HOST=$(cat /ssh/host)
+IS_LOCALHOST=$([ "$SSH_HOST" == "localhost" ] && echo true || echo false)
```

Comment:
instead of using a true/false string, consider using set/has value v.s. unset/empty string. That way a "true" set value is detected with:
```
if [[ $IS_LOCALHOST ]] ...
```
There as the "false" unset value is detected with:
```
if [[ -z $IS_LOCALHOST ]]
```
or
```
if ! [[ $IS_LOCALHOST ]]
```